server/api/internal/search/reindex.post.ts

```
@@ -0,0 +1,56 @@
+import { getHeader } from 'h3'
+import { cloudflareEnv, jsonResponse } from '~/server/utils/api-response'
+import { rebuildPlatformKnowledgeIndex } from '~/server/utils/public-search'
+
+async function secretsMatch(expectedSecret: string, providedSecret: string) {
+  const encoder = new TextEncoder()
+  const subtle = crypto.subtle as SubtleCrypto & {
+    timingSafeEqual?: (left: BufferSource, right: BufferSource) => boolean
```

ESLint

[error] 'left' is defined but never used. Allowed unused args must match /^_/u.

(no-unused-vars)